src/msix/pack/ContentTypeWriter.cpp

```diff
@@ -44,17 +44,24 @@ namespace MSIX {
     {
         auto percentageEncodedName = Encoding::EncodeFileName(name);
 
+        auto lastDir = percentageEncodedName;

```

Comment:
>lastDir [](start = 13, length = 7)

Typically this would be called "filename", as it is not a directory but the leaf part, aka the file name.